packages/@webex/plugin-meetings/src/meeting/muteState.ts

```diff
@@ -150,32 +150,34 @@ export class MuteState {
    * @param {Boolean} [mute] true for muting, false for unmuting request
    * @returns {void}
    */
-  public handleLocalStreamMuteStateChange(meeting?: object, mute?: boolean) {
+  public handleLocalStreamMuteStateChange(meeting?: any) {
     if (this.ignoreMuteStateChange) {
       return;
     }
+
+    // either user or system may have triggered a mute state change, but localMute should reflect both
+    let newMuteState: boolean;
+    let userMuteState: boolean;
+    let systemMuteState: boolean;
+    if (this.type === AUDIO) {
+      newMuteState = meeting.mediaProperties.audioStream?.muted;
+      userMuteState = meeting.mediaProperties.audioStream?.userMuted;
+      systemMuteState = meeting.mediaProperties.audioStream?.systemMuted;
+    } else {
+      newMuteState = meeting.mediaProperties.videoStream?.muted;
+      userMuteState = meeting.mediaProperties.videoStream?.userMuted;
+      systemMuteState = meeting.mediaProperties.videoStream?.systemMuted;
+    }
+
     LoggerProxy.logger.info(
-      `Meeting:muteState#handleLocalStreamMuteStateChange --> ${this.type}: local stream new mute state: ${mute}`
+      `Meeting:muteState#handleLocalStreamMuteStateChange --> ${this.type}: local stream new mute state: ${newMuteState} (user mute: ${userMuteState}, system mute: ${systemMuteState})`
     );
 
-    this.state.client.localMute = mute;
+    this.state.client.localMute = newMuteState;
 
     this.applyClientStateToServer(meeting);
   }
 
-  /**
-   * Applies the current mute state to the local stream (by enabling or disabling it accordingly)
-   *
-   * @public
-   * @param {Object} [meeting] the meeting object
-   * @param {ServerMuteReason} reason - reason why we're applying our client state to the local stream
-   * @memberof MuteState
-   * @returns {void}
-   */
-  public applyClientStateLocally(meeting?: any, reason?: ServerMuteReason) {
-    this.muteLocalStream(meeting, this.state.client.localMute, reason);
-  }
```

Comment:
Why wouldn't we need this anymore? Particularly when it's a publicly exposed method? We won't know which customer might still be using it

Reply:
We pulled it out because SDK wasn't using this function anymore and the web client wasn't using it either. But if a customer might still be using it, it is an argument to put it back in.